app/src/services/version.js

```diff
@@ -301,10 +306,16 @@ const service = {
           .whereNot({ 'id': updated[0].id })
           .orderBy('createdAt', 'desc');
 
-        if (!sq.some(v => v.isLatest).length) {
+        if (sq.length && !sq.some(v => v.isLatest).length) {
           await Version.query(trx)
-            .update({ 'isLatest': true, 'objectId': objectId, updatedBy: userId })
-            .where({ 'id': sq[0]?.id, 'objectId': objectId });
+            .update({
+              'isLatest': true,
+              'objectId': objectId,
+            })
+            .where({
+              'id': sq[0]?.id,
+              'objectId': objectId
+            });
```

Comment:
Do we need to be re-indenting the inner objects when they're just two attributes?

Reply:
good call. i've now condensed a few of these